ISSUE_TEMPLATE.md

```diff
@@ -1 +1,47 @@
+<!--
+Please use the below format to report the issue in a better way.
+To check any option, replace the space between the "[" and "]" with a "x".
+-->
 
+#### Nature of Issue ?
+
+- [ ] Found a Bug
+- [ ] Existing Feature Enhancement
+- [ ] New Feature Request
+
+#### Most Appropriate Sub Area ?
+
+- [ ] Color
+- [ ] Core
+- [ ] Events
+- [ ] Image
+- [ ] IO
+- [ ] Math
+- [ ] Typography
+- [ ] Utilities
+- [ ] WebGL
+
+<!--
+	Fill the following sections as suitable, and delete the part which is not applicable.
+-->
```

Comment:
I think we can remove this comment, and add the note to delete the remaining parts to the comment at the top